### ISSUE TITLE
force-color ignored in the config file

### ISSUE THREAD

**Issue by 0xjac** [ansible/ansible-lint#1078]
##### Summary

The option `force-color` in the config file is ignored.
Passing `--force-color` on the command line works fine.

I have tried multiple variants:
- `force-color`
- `force_color`
- `forcecolor`
- `colored`

None of them works.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible 2.10.1
  config file = $HOME/.ansible.cfg
  configured module search path = ['$HOME/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.8/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.5 (default, Sep  5 2020, 10:50:12) [GCC 10.2.0]

ansible-lint --version
ansible-lint 4.3.5

```

- ansible installation method: OS package
- ansible-lint installation method: OS package

##### OS / ENVIRONMENT

```console
$ uname -srvmpio
Linux 5.8.11-1-MANJARO #1 SMP PREEMPT Wed Sep 23 14:35:40 UTC 2020 x86_64 unknown unknown GNU/Linux

$ lsb_release -drcs
"Manjaro Linux" 20.1.2 Mikah
```

##### STEPS TO REPRODUCE

Run `ansible-lint` on a non interactive terminal such as Gitlab CI/CD,
with a playbook containing some error (like a line longer than 160 chars) to trigger a warning.
Use the following `.ansible-lint` config file:
```yaml
verbosity: 2
force-color: true
```

```console (paste below)
ansible-lint

# non-colored output
```

##### Desired Behaviour

The output should be colored.

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behaviour

The output is not colored.

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:
- playbook
  ```yaml
  - name: Hostname is set                                                                                                                                        too long
    become: yes
    hostname:
      name: "{{ hostname }}"
  ```
- output of running ansible-lint
  Can't be shown, the output is the same, it's just not colored...
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`
   No strack trace

```paste below
DEBUG    Logging initialized to level 10
DEBUG    Options: Namespace(colored=False, config_file=None, display_relative_path=True, exclude_paths=[], format='rich', listrules=False, listtags=False, parseable=False, parseable_severity=False, playbook=['roles'], quiet=False, rulesdir=[], skip_list=[], tags=[], use_default_rules=False, verbosity=2, warn_list=['experimental'])
DEBUG    Loading rules from /usr/lib/python3.8/site-packages/ansiblelint/rules
DEBUG    Examining roles/tasks/hostname.yml of type tasks
You can skip specific rules or tags by adding them to your configuration file:  

┌──────────────────────────────────────────────────────────────────────────────┐
│# .ansible-lint                                                               │
│warn_list:  # or 'skip_list' to silence them completely                       │
│  - '204'  # Lines should be no longer than 160 chars                         │
└──────────────────────────────────────────────────────────────────────────────┘
204 Lines should be no longer than 160 chars
roles/tasks/hostname.yml:1
- name: Hostname is set                                                                                                                                        too long


```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by ssbarnea** [comment]
AFAIK there is no documentation stating that this option can be specific in the config file, so basically there is no bug. 

I am going to close this ticket but feel free to make a PR that enables this option and documents it. Keep in mind that command line is always supposed to override options from config fix.
